package.json

```diff
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "http-errors": "^1.7.1",
-    "humanize-duration": "^3.17.0",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
Are the changes to package-lock.json missing?

Reply:
I see `package-lock.json` is blacklisted in `.gitignore`. Should be removed from there and comitted.

Reply:
I'll do it now